tensorflow/python/grappler/auto_mixed_precision_test.py

```diff
@@ -394,6 +410,55 @@ def test_conv_bn(self):
         self.assertEqual(num_to_fp32, 1)  # After FusedBatchNormV3:0
         self.assertAllClose(output_val_ref, output_val, atol=1e-3, rtol=1e-3)
 
+  @test_util.run_deprecated_v1
+  @test_util.disable_xla('This test does not pass with XLA')
+  def test_conv3d_bn(self):
```

Comment:
This test will fail in the GitHub tests, as they don't yet run with cuDNN 7.6.2.

You can delete them or disable them with [`@unittest.skip`](https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures). Or you can wait until we start running with cuDNN >= 7.6.2.